cmd/project.go

```diff
@@ -804,6 +806,81 @@ Examples:
 	},
 }
 
+var projectRollbackDeploymentCmd = &cobra.Command{
+	Use:   "rollback-deployment",
```

Comment:
having this under project is a bit weird when deploy isnt.

i think we should put it under cloud (like deploy) and name is either rollback or undeploy (and alias to other). and alias to root.

so i can just do

agentuity undeploy

agentuity undeploy --tag name